### ISSUE TITLE
text embedding模型量化

### ISSUE THREAD

**Issue by Data-Adventure** [airockchip/rknn-toolkit2#289]
我有一个text embedding模型，目前我使用非量化的fp精度，可以导出.rknn模型，但是还想进一步进行int8或者w8a8量化。请问该如何进行呢？

**Comment by yuyun2000** [comment]
建议放弃

**Comment by yuyun2000** [comment]
不过我也试过，大多数层都是正常的，你要尝试可以输入npy量化试试，也可以只量化中间的linear看看

**Comment by Data-Adventure** [comment]
> 不过我也试过，大多数层都是正常的，你要尝试可以输入npy量化试试，也可以只量化中间的linear看看

谢谢解答，我会试试看输入npy量化的，不过当前我遇到的问题是，npy如何组织？因为这个模型输入分别为：input_ids、attention_mask、token_type_ids，我需要将语句预处理成这三个，然后再组成npy么？

**Comment by yuyun2000** [comment]
是的，支持多npy输入进行量化，具体看文档示例

**Comment by Data-Adventure** [comment]
> 是的，支持多npy输入进行量化，具体看文档示例

好的，感谢支持，我会实验一下看看结果

**Comment by hebangwen** [comment]
打扰一下二位，想问一下带有 attention-mask 的 text embedding 模型转为 RKNN 之后效果如何？我导出了一个 [InternVL2 LLM 部分的 FP16 prefill.rknn 模型](https://github.com/sophgo/LLM-TPU/blob/main/models/InternVL2/compile/export_onnx.py)，发现首 token 输出错误，并且 token-0 之外的 hidden-states 相似度不高。
